week-1/exercises/paths.js

```diff
@@ -16,3 +16,7 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+
+let dirPath=filePath.slice(0,-8);
```

Comment:
This gets you to the right answer. However, you have a hardcoded number '-8' here. This can appear as a '[magic number](https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings)' to readers of the codebase. How might you change this to avoid hardcoding the number? Is there a variable that's defined above that might help?

Reply:
it'd be -base.length, I have changed the code accordingly